pkg/package/package.go

```diff
@@ -385,7 +452,32 @@ func (kclPkg *KclPkg) ValidateKpmHome(kpmHome string) error {
 	return nil
 }
 
+// GetPkgFullName returns the full name of package.
+// The full name is "<pkg_name>-<pkg_version>",
+// <pkg_name> is the name of package.
+// <pkg_version> is the version of package
+func (kclPkg *KclPkg) GetPkgFullName() string {
+	return kclPkg.modFile.Pkg.Name + "-" + VERSION_PREFFIX + kclPkg.modFile.Pkg.Version
+}
+
 // GetPkgName returns name of package.
 func (kclPkg *KclPkg) GetPkgName() string {
 	return kclPkg.modFile.Pkg.Name
 }
+
+// GetPkgTag returns version of package.
+func (kclPkg *KclPkg) GetPkgTag() string {
+	return kclPkg.modFile.Pkg.Version
+}
+
+// GetPkgTagForOci returns version of package in OCI format.
+// The version of a package is "0.0.1".
+// The version of a package in OCI format is "v0.0.1"
+func (kclPkg *KclPkg) GetPkgTagForOci() string {
+	return "v" + kclPkg.GetPkgTag()
```

Comment:
Here, a defensive judgment is needed to determine whether `kclPkg.GetPkgTag()` starts with the "v" character and maintains the idempotence of the function `GetPkgTagForOci`.

Reply:
fixed.